projects/addon-editor/directives/tiptap-editor/tiptap-editor.service.ts

```diff
@@ -160,6 +161,13 @@ export class TuiTiptapEditorService extends TuiEditor {
         return this.editor.isActive(name);
     }
 
+    isActive$(nameOrAttributes: Record<string, string> | string): Observable<boolean> {
+        return this.stateChange$.pipe(
+            map(() => this.editor.isActive(nameOrAttributes)),
```

Comment:
`this.isActive` and lets extend its attribute type with `Record`

Reply:
fixed